elements/pfe-styles/src/pfe-typography-classes.scss

```diff
@@ -1,14 +1,15 @@
 @import "../../pfe-sass/pfe-sass";
 
 // TITLES
-@each $size in (6xl, 5xl, 4xl, 3xl, 2xl, xl, lg, md, sm, xs) {
+@each $size in ("6xl", "5xl", "4xl", "3xl", "2xl", "xl", "lg", "md", "sm", "xs") {
```

Comment:
The quotes only matter on the items that start with a number.

Reply:
yeah I figured consistency was easier